definitions/npm/react-native-thumbnail-video_v0.x.x/flow_v0.77.0-/react-native-thumbnail-video_v0.x.x.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+type PreviewTypes = $ReadOnly<{|
```

Comment:
These are now defined in global scope. I guess this is not intentional? These can be moved inside library definition.